backend/inspirehep/records/marshmallow/fields/nested_without_empty_objects.py

```diff
@@ -5,16 +5,12 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
-from inspire_dojson.utils import strip_empty_values
 from marshmallow import fields, missing
 
 
-class NestedWithoutEmptyObjects(fields.Nested):
+class NestedObjects(fields.Nested):
```

Comment:
I would rename it to `NestedField` or something like that since we don't have Empty Objects inside anymore ;)